deployment/README.md

```diff
@@ -1 +1,47 @@
 # Deployment
+
+## Requirements
+
+* Helm
+* [helm-diff plugin](https://github.com/databus23/helm-diff)
+* helmfile
+
+## Helm
+
+Move on to your helmfile environment, for example local and then apply your changes:
+> cd /helmfile/local
+>
+> helmfile -f helmfile.yaml apply
+>
+
+### Folder Structure
+```
+.
+├── charts
+│   └── catalog-rest-api
+│       ├── Chart.yaml
+│       ├── templates
+│       │   ├── deployment.yaml
+│       │   ├── _helpers.tpl
+│       │   └── service.yaml
+│       └── values.yaml
+├── helmfiles
+│   ├── dev
+│   │   └── helmfile.yaml
+│   ├── local
+│   │   ├── helmfile.yaml
+│   │   └── values.yaml
+│   └── prod
+├── README.md
+└── values
+    ├── dev
+    │   └── catalog-rest-api
+    │       └── values.yaml
+    └── replica-values.yaml
```

Comment:
I think that having such a structure can easily be out-to-dated. Maybe it's easier to have the main directories only.

Reply:
Yes, maybe in the sort term this diagram will be removed.